cni-plugin/pkg/dataplane/linux/dataplane_linux.go

```diff
@@ -76,14 +76,14 @@ func (d *linuxDataplane) DoNetworking(
 	}
 
 	err = ns.WithNetNSPath(args.Netns, func(hostNS ns.NetNS) error {
+		la := netlink.NewLinkAttrs()
```

Comment:
I'm worried that using the constructor masks the issue and that we may not realize what values we are intending to set here for `TxQLen` and `NetNsID`. Maybe we should add a comment here stating that this constructor should also be setting those values so that we will use the kernel defaults?

Reply:
Hmmm idk, the library specifically says to use the constructor or set all the values properly. We would have to comment every usecase. I wish there was no accesses to the individual items and then there would be no such an issue but to use the constructor :man_shrugging: I think this is a great catch! Thanks @radixo !